app/views/audios/new.html.erb

```diff
@@ -1,5 +1,10 @@
+<script>
+    var pss_create_asset_path = '<%= source_audios_path() %>';
+    var pss_asset_type = 'audio';
+</script>
```

Comment:
It might be cleaner to put this javascript in the `<head>` rather than the `<body>` (see http://stackoverflow.com/a/3438707)